Wikipedia/Code/WMFFeedContentSource.m

```diff
@@ -328,8 +346,7 @@ - (nullable WMFContentGroup *)featuredForDate:(NSDate *)date inManagedObjectCont
 }
 
 - (nullable WMFContentGroup *)pictureOfTheDayForDate:(NSDate *)date inManagedObjectContext:(NSManagedObjectContext *)moc {
-    //NOTE: POTDs are the same across languages so we do not not want to constrain our search by site URL as this will cause duplicates
-    return (id)[moc groupOfKind:WMFContentGroupKindPictureOfTheDay forDate:date];
+    return (id)[moc groupOfKind:WMFContentGroupKindPictureOfTheDay forDate:date siteURL:self.siteURL];
```

Comment:
This is the other main logic change (part 2 of 2). We use the site URL (in the same way as other content) and the duplicate logic is now handled as part of the `save` method.

_N.B. All other changes to this file are just clang-format changes_